vignettes/Using_predictInterval.Rmd

```diff
@@ -201,9 +201,9 @@ effects as fully-known (that is, with an effectively 0 variance.) This is done
 using the `ignore.fixed.effects` argument.
 
 ```{r ShowFat2, message=FALSE, warning=FALSE}
-predictInterval(ifmodel, ais[1,], include.resid.var=0, ignore.fixed.terms = 1) #predict the average body fat for a group of 196cm female baseball players, taking the global intercept (mean body fat) as fully known
-predictInterval(ifmodel, ais[1,], include.resid.var=0, ignore.fixed.terms = "(Intercept)") #Same as above
-predictInterval(ifmodel, ais[1,], include.resid.var=0, ignore.fixed.terms = 1:2) #as above, taking the first two fixed effects (intercept and height effect) as fully known
+predictInterval(fatmodel, ais[1,], include.resid.var=0, ignore.fixed.terms = 1) #predict the average body fat for a group of 196cm female baseball players, taking the global intercept (mean body fat) as fully known
```

Comment:
Hi @jamesonquinn  -- where does the `ais` dataset come from? Need to include it in the full unit test suite and I can't seem to find it.